tests/auth_tests/test_auth_backends.py

```diff
@@ -803,6 +814,130 @@ def test_clean_credentials_sensitive_variables(self):
             status_code=500,
         )
 
+    @override_settings(
+        ROOT_URLCONF="django.contrib.auth.urls",
+        AUTHENTICATION_BACKENDS=["auth_tests.test_auth_backends.TypeErrorBackend"],
+    )
+    def test_login_process_sensitive_variables(self):
+        try:
+            self.client.post(
+                reverse("login"),
+                dict(username="testusername", password=self.sensitive_password),
+            )
+        except TypeError:
+            exc_info = sys.exc_info()
+
+        rf = RequestFactory()
+        response = technical_500_response(rf.get("/"), *exc_info)
+
+        self.assertNotContains(response, self.sensitive_password, status_code=500)
+        self.assertContains(response, "TypeErrorBackend", status_code=500)
+
+        # AuthenticationForm.clean().
+        self.assertContains(
+            response,
+            '<tr><td>password</td><td class="code">'
+            "<pre>&#39;********************&#39;</pre></td></tr>",
+            html=True,
+            status_code=500,
+        )
+
+        # Client.post().
+        self.assertContains(
+            response,
+            '<tr><td>data</td><td class="code">'
+            "<pre>&#x27;********************&#x27;</pre></td></tr>",
+            html=True,
+            status_code=500,
+        )
```

Comment:
Since rebasing, this seems to be failing but I am not sure why - can you have a look?

Reply:
Ahh, sorry about that. `Client.post` and `RequestFactory.post` are functions used only during testing (the ones I was talking about earlier), because we're not hiding them anymore, shouldn't we remove these checks?

Reply:
Sounds good 👍